airflow/config_templates/default_airflow.cfg

```diff
@@ -49,6 +49,13 @@ s3_log_folder =
 # Logging level
 logging_level = INFO
 
+# Size of logging buffer
+logging_buffer_size = 4096
+# Logging level minimum to flush buffer before full
+# See python doc, DEBUG=10,INFO=20,WARN=30,ERROR=40
```

Comment:
Would be good to replace "python doc" with an actual link, it's not 100% clear where to look.